README.md

```
@@ -34,114 +34,94 @@
+"Test connection to staging-server"
+"Discover capabilities on all hosts"
+"Update the compose path for production-1 to /opt/stacks"
+"Remove the old-server host from my configuration"
 ```
 
 ### Tool 2: `docker_container`
-Control containers across all your hosts.
+Consolidated Docker container management tool.
 
 **Actions:**
 - `list` - List containers on a host
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)